### ISSUE TITLE
Helm 3 - Installer chart

### ISSUE THREAD

**Issue by iMax-pp** [pegasystems/pega-helm-charts#43]
With Helm 3, the installer chart (with parameter `actions.execute` being `install` or `upgrade`) cannot be installed. We get the following error:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [unknown object type "nil" in ConfigMap.data.DB2ZOS_UDF_WLM, unknown object type "nil" in ConfigMap.data.DISTRIBUTION_KIT_URL, unknown object type "nil" in ConfigMap.data.JDBC_CUSTOM_CONNECTION]
```
It seems to be similar to issue #34 but on a different chart.

Could you please take a look into it?

**Comment by dcasavant** [comment]
Merged as part of #45.  This will be published in the next release.

**Comment by charoensri** [comment]
Hi We tried the Installer image, but the upgrade action was an invalid action ie. it is commented out.  With the "install" action, it assumes a clean schema which we cannot use to the upgrade. Is upgrade action still supported and how?
